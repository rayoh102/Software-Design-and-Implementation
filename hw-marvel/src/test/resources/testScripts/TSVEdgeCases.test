# Tests if edge cases like invalid character or no path are handled properly in a custom TSV

# Load Graph
LoadGraph custom sampleData.tsv

# Find an invalid start char

FindPath custom Her1 Hero3

# Find an invalid end char

FindPath custom Hero1 Her6

# Find two invalid chars

FindPath custom DNE1 DNE2

# Find valid chars with no path

FindPath custom Hero7 Hero4