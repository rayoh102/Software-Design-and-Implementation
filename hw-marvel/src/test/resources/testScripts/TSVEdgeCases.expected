# Tests if edge cases like invalid character or no path are handled properly in a custom TSV

# Load Graph
loaded graph custom

# Find an invalid start char

unknown character Her1

# Find an invalid end char

unknown character Her6

# Find two invalid chars

unknown character DNE1
unknown character DNE2

# Find valid chars with no path

path from Hero7 to Hero4:
no path found